tests/console/yast2_proxy.pm

```diff
@@ -50,11 +50,19 @@ sub empty_field {
     }
 }
 
+sub post_fail_hook {
+    my ($self) = shift;
+    $self->SUPER::post_fail_hook;
+    upload_logs("/etc/squid/squid.conf");
+    upload_logs("/var/log/squid/access.log");
+    upload_logs("/var/log/squid/proxy_cache.log");
+    upload_logs("/var/log/squid/proxy_store.log");
+}
+
 sub run {
     select_console 'root-console';
     if (is_sle && sle_version_at_least('15')) {
-        my $ret = zypper_call('in squid', exitcode => [0, 104]);
-        return record_soft_failure 'bsc#1056793' if $ret == 104;
+        my $ret = zypper_call('in squid');
```

Comment:
Let's remove variable, as not used anymore

Reply:
Fixed now.